File: CheckPrime/checkPrime.js
const checkPrime =(n)=>{
    if(n<=1){
        return false;
    }
    let sq=Math.sqrt(n);
    let count=0;
    for(let i=1;i<=sq;i++){
        if(n%i==0){
            count++;
            if(n/i !==i){
                count++;
            }
        }


    }
    if(count==2){
        return true;
    }
    return false;


}
let n=14;
console.log(checkPrime(n));

File: CheckPrime/checkPrime.py
import math
def checkPrime(n):
    
    cnt=0
    
    sq=math.sqrt(n)
    for i in range(1,int(sq)+1):
        if n%i==0:
            cnt+=1
            if n//i != i:
                cnt+=1
        
    if cnt==2:
        return True
    return False
    
n=14
print(checkPrime(n))